core/src/main/java/fj/data/List.java

```diff
@@ -1497,7 +1546,7 @@ private void tail(final List<A> tail) {
    * @return A list of the given value replicated the given number of times.
    */
   public static <A> List<A> replicate(final int n, final A a) {
-    return n <= 0 ? List.<A>nil() : replicate(n - 1, a).cons(a);
+    return n <= 0 ? List.<A>nil() : List.<A>replicate(n - 1, a).cons(a);
   }
 
```

Comment:
@mperry foldMOption in here is nothing but traverseOption. change in replcate makes IntelliJ happy.